bq_workers/gitlab_parser/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Use the official Python image.
+# https://hub.docker.com/_/python
+FROM python:3.7
+
+# Allow statements and log messages to immediately appear in the Cloud Run logs
+ENV PYTHONUNBUFFERED True
+
+# Copy application dependency manifests to the container image.
+# Copying this separately prevents re-running pip install on every code change.
+COPY requirements.txt .
+
+# Install production dependencies.
+RUN pip install -r requirements.txt
```

Comment:
is it just 'pip' in the python 3.7 image and not pip3?

Reply:
Yes!  Since we're using the python3 image, we don't have to worry about `pip` defaulting to python2.  If it were outside of the Docker container, we would want to specify pip3, b/c otherwise it's dependent on developer environment and could install packages to the wrong python version.

Reply:
This is how we do it all the python-docs-samples. eg: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/run/pubsub/Dockerfile